plugin/src/main/java/org/godotengine/openxr/vendors/GodotOpenXR.kt

```diff
@@ -43,6 +43,13 @@ abstract class GodotOpenXR(godot: Godot?) : GodotPlugin(godot) {
 		protected val TAG = GodotOpenXR::class.java.simpleName
 
 		init {
+			try {
+				Log.v(TAG, "Loading openxr_loader library")
+				System.loadLibrary("openxr_loader")
```

Comment:
Let's keep them in the same `try..catch` block. With the current logic, the first `loadLibrary` could fail, and we would still attempt the second `loadLibrary`. By keeping them together, if one fails they both fail.